assets/css/components/_brand.scss

```diff
@@ -7,33 +7,18 @@
 [class^="gpii-ext-theme"] .logo path,
 [class^="gpii-ext-theme"] .logo circle {
   fill: none;
-  stroke: var(--logo-stroke);
+  stroke: var(--logo-stroke, currentColor);
   stroke-linecap: round;
   stroke-width: 0.1875rem;
 }
 
-.fl-theme-wb,
-.gpii-ext-theme-wb {
-  --logo-stroke: #{$white};
-}
-
-.fl-theme-yb,
-.gpii-ext-theme-yb {
-  --logo-stroke: #{$yellow};
-}
-
-.fl-theme-bw,
-.gpii-ext-theme-bw,
-.fl-theme-by,
-.gpii-ext-theme-by,
 .fl-theme-bbr,
 .gpii-ext-theme-bbr {
   --logo-stroke: #{$black};
 }
 
-.fl-theme-gw .logo-element,
-.gpii-ext-theme-gw .logo-element,
-.fl-theme-gd .logo-element,
-.gpii-ext-theme-gd .logo-element {
+.fl-theme-lgdg,
+.gpii-ext-theme-gw,
+.gpii-ext-theme-gd {
```

Comment:
I mentioned this in a comment, but it was after you made this commit. UIO also supports `fl-theme-gw` and `fl-theme-gd`. The `fl-theme-lgdg` is a different theme that isn't used in UIO+ but is available in UIO. It is a binary theme though so you won't need to address it separately here, but you will need to address the other two classes I mentioned.